Hai Reveal conversations: in A14 (treaty negotiations) more player engagement and no text walls (#7594)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

	on offer
		log `The Hai have threatened the Republic if suitable action isn't taken. Required to escort civilians out of the possible danger zone.`
		conversation
			`Upon docking at <origin>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`

> 			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as Cabinet is blocking any progress.`

Author: Done.

			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns to you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed, and we've gathered to prepare you for the meeting."`
			label questions
			choice
				`	"I'm ready to mediate. Let's assemble the heads of state."`

> 					goto "begin the meeting"

Reviewer: Typically the "I'm done" choice goes last with choice branches like this.

Author: Corrected.

				`	"Can the Hai really pass through, with the Republic Navy blocking their way?"`

> 					goto "navy blocks hai"

Reviewer: I prefer one word labels, but I'd say it's fine to leave this if you don't want to mess with it.

Author: The labels are all one word now.

				`	"What if we let them through, to attack the pirate planets?"`
					goto "let hai through"
				`	"What if we solve the kidnapping problem instead of the Hai doing it?"`
					goto "hai concerns"
				`	"Who in Cabinet is blocking progress?"`

> 					goto "who is blocking"

Author: I didn't want to wait another two years for that feature, so I got it working in #7629, but I don't know how to add tests.

> 					goto "who is blocking"

Author: Waiting to hear back from original PR author on how to proceed.

> 					goto "who is blocking"

Author: My fixes to #5440 are here: https://github.com/SolraBizna/endless-sky/pull/1

> 					goto "who is blocking"

Author: Your wish has been granted. There is no looping dialog, and two of the questions are now follow-up questions. You won't ask about letting the Hai attack pirates, or having the Navy solve the problem, until you ask if the Navy can block the Hai from passing through. The questions are arranged to ensure you'll ask them all if you just keep hitting "enter."

			label formality
			`	Teeneep explains, "I speak as the voice of the Hai and with the full authority of the Elder Council. Thousands of our people have been kidnapped over the past few decades, and they still largely reside under the servitude of various pirate factions that the Republic allows to continue. Even worse, your authorities have allowed numerous Hai to be slipped past on slaver ships, without attempting rescue because of the no boarding policy your forces employ, even on disabled and potentially doomed ships. We demand the return of our people and an immediate end to the policy of letting slaves die or escape without attempting boarding and rescue."`
			choice
				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleges from the other side of the wormhole can help you solve this problem. President? Chancellor?"`
					goto formality2
				`	"Thank you, Teeneep. President, Chancellor, please explain the piracy situation from the human point of view."`

> 			`	The President seems surprised by the informal tone of your question, as if you just made a faux pah. The Chancellor looks more at ease than before, and perhaps a little amused.`

Reviewer 2: ```suggestion
			`	The President seems surprised by the informal tone of your question, as if you just made a faux pas. The Chancellor looks more at ease than before, and perhaps a little amused.`
```

			label formality2
			`	The President is the first to respond. "The problem of piracy in human space is a difficult one and cannot be solved overnight. The pirates are well-entrenched in their planets, and fighting them is a long and brutal struggle. With our navies distracted fighting one another, our recent war has left them stronger than ever."`
			`	The Chancellor states, "However, the Free Worlds has made great strides reducing piracy by befriending the peoples of the anarchist worlds. We are discussing similar approaches with the Republic. Perhaps in another few years, piracy will be entirely eliminated."`
			choice

> 				`	"With respect Chancellor, what you propose would take years to resolve a problem that is an immediate concern. How does the Hai representative respond?"`

Reviewer 2: ```suggestion
				`	"With respect, Chancellor, what you propose would take years to resolve a problem of immediate concern. How does the Hai representative respond?"`
```

					goto "keep teeneep calm"
				`	"Is this an acceptable proposal, Teeneep?"`

> 			`	Teeneep looks perturbed while she holds back her anger and resentment. The other heads of state don't seem notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with Hai.  "Of course, not," she says.`

Reviewer 2: ```suggestion
			`	Teeneep looks perturbed, but she makes an effort to hold back her anger and resentment. The other heads of state don't seem notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with Hai.  "Of course not," she says.`
```

> 			`	Teeneep looks perturbed, but she makes an effort to hold back her anger and resentment. The other heads of state don't seem notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with Hai.  "Of course not," she says.`

Reviewer 2: ```suggestion
			`	Teeneep looks perturbed, and has to make an effort to hold back her anger and resentment. The other heads of state don't seem notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with the Hai. "Of course not," she says.`
```

> 			`	Teeneep looks perturbed, and has to make an effort to hold back her anger and resentment. The other heads of state don't seem notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with the Hai. "Of course not," she says.`

Reviewer 2: ```suggestion
			`	Teeneep looks perturbed, and has to make an effort to hold back her anger and resentment. The other heads of state don't seem to notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with the Hai. "Of course not," she says.`
```

			label "keep teeneep calm"
			`	Teeneep responds plainly, "We are a peaceful people by nature, but these pirate attacks have gone too far. We support a solution that benefits the innocent inhabitants of the pirate worlds. However, there must be immediate action, not years of delay."`
			choice
				`	"Surely a group of enlightened human leaders can propose an alternative?"`
				`	"Are there any counter-proposals?"`
					goto "keep president happy"

> 			`	The President seems genuinely annoyed by being patronized, while the Chancellor smirks. You get a furry little smile from Teeneep.`

Reviewer 2: ```suggestion
			`	The President seems genuinely irritated by being patronized, but the Chancellor smirks. You get a furry little smile from Teeneep.`
```

> 			`	The President seems genuinely irritated by being patronized, but the Chancellor smirks. You get a furry little smile from Teeneep.`

Reviewer 2: ```suggestion
			`	The President seems genuinely irritated at being patronized, but the Chancellor smirks. You get a furry little smile from Teeneep.`
```

			label "keep president happy"

> 			`	"We have a counter-proposal," the President says. "We cannot afford to change our policy in totality, it would cost too many lives. However, we propose the following: The Free Worlds and Republic will establish a customs service on Mountaintop. Hai visiting human space will check in with us there. Our intelligence agencies will then track their movements in order to see to their safety as best we can. They'll also be given contacts on every human planet that they can go to for help. In the meantime, we will place extensive surveillance fleets at Ultima Thule to detect any unapproved Hai travel. We will do our best to disable and capture ships with kidnapped Hai, no matter the risk to Navy lives."`

Reviewer: We don't capitalize after semicolons and colons.
```suggestion
			`	"We have a counter-proposal," the President says. "We cannot afford to change our policy in totality, it would cost too many lives. However, we propose the following: the Free Worlds and Republic will establish a customs service on Mountaintop. Hai visiting human space will check in with us there. Our intelligence agencies will then track their movements in order to see to their safety as best we can. They'll also be given contacts on every human planet that they can go to for help. In the meantime, we will place extensive surveillance fleets at Ultima Thule to detect any unapproved Hai travel. We will do our best to disable and capture ships with kidnapped Hai, no matter the risk to Navy lives."`
```

			label "no human response"
			`	Teeneep takes a moment to respond. "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
			`	The President replies, "The Minister for Foreign Affairs has been keeping Cabinet apprised of the major details and proposals reported back by Deputy Minister Yang. Under the circumstances we believe it would be appropriate for us to attempt a joint effort to address your concerns. We agree to making this effort against one target, and to reassessing the situation afterwards depending upon what is discovered in that process. The Republic representatives there with you now will work with you to identify that target and plan the action accordingly."`
			`	The smoothness with which this goes down gives you the sense that the main points of this were all provisionally agreed to already, with only a little wiggle room left to be settled in this conversation.`
			`	Sayari steps up and whispers something to Teeneep who then seems to think for a moment before returning to the screens. "This pathway is acceptable."`
			choice

> 				`	(End the meeting.)`

Reviewer: Another empty choice, as I explained in #7566.

Author: The player has three ways to end the conversation now.

	on offer
		log `The Hai have threatened the Republic if suitable action isn't taken. Required to escort civilians out of the possible danger zone.`
		conversation

> 			`Upon docking at <origin>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`

Reviewer 3: ```suggestion
			`Upon docking at <origin>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`
```

					goto formality
				`	"Teeneep, please explain the problem to the human leaders. I'm sure we can come to a compromise that will suit everyone."`
			`	Teeneep responds, "Thank you for mediating these proceedings, Captain <last>."`
			`	Both human leaders seem put off by the level of familiarity you just expressed, but Teeneep looks as if you took a world of stress off her shoulders.`
			label formality
			`	Teeneep explains, "I speak as the voice of the Hai and with the full authority of the Elder Council. Thousands of our people have been kidnapped over the past few decades, and they still largely reside under the servitude of various pirate factions that the Republic allows to continue. Even worse, your authorities have allowed numerous Hai to be slipped past on slaver ships, without attempting rescue because of the no boarding policy your forces employ, even on disabled and potentially doomed ships. We demand the return of our people and an immediate end to the policy of letting slaves die or escape without attempting boarding and rescue."`
			choice

> 				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleges from the other side of the wormhole can help you solve this problem. President? Chancellor?"`

Reviewer 2: ```suggestion
				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleagues from the other side of our wormhole can help you solve this problem. President? Chancellor?"`
```

> 				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleagues from the other side of our wormhole can help you solve this problem. President? Chancellor?"`

Reviewer 2: ```suggestion
				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleagues from the other side of the wormhole can help you solve this problem. President? Chancellor?"`
```
Sorry, I edited this afterwards and it didn't change back to "the".

		log `The Hai have threatened the Republic if suitable action isn't taken. Required to escort civilians out of the possible danger zone.`
		conversation

> 			`Upon docking at <planet>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`

Reviewer 4: 'Upon docking at <planet>' should be 'Upon docking at <origin>'; playthrough had me confused when I landed on Mountaintop and the conversation said I was on Sunracer. (Sorry I don't know how to contribute changes yet)

Reviewer 2: When you're commenting, you can click the ± inside a page to get the line you want to change. I'll do this one for you. 
```suggestion
			`Upon docking at <origin>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`
```
I am pretty sure this was changed earlier in another PR.

Author: That does sound a bit off. Yes, I will fix that. Thank you for noticing.

Author: > I am pretty sure this was changed earlier in another PR.

I've been merging master to this branch frequently, but I have so many text changes that some fixes may have gotten lost in the conflict resolution.

Author: I found another planet-origin mixup in part 1, Hai Tour [5] Tour Ends, which I've fixed in #7566. Thanks for getting me to check on those.

> 			`	Sayari is intercepted by a Hai functionary and led away as you reach Xilin and the other diplomats, also joined by Danforth. Xilin looks at you all. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as the cabinet is blocking any progress.`

Reviewer 2: ```suggestion
			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as the cabinet is blocking any progress.`
```

Reviewer 2: Maybe 'Xilin looks at you all' could go on the next line.

> 			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as the cabinet is blocking any progress.`

Reviewer 2: ```suggestion
			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as Cabinet is blocking any progress.`
```

Author: I just changed "the Cabinet" and "the cabinet" to "Cabinet" several places. It seems quite odd though, to have a group that is simply called "Cabinet."

Author: I've changed it to "the Cabinet" everywhere, since that seems more natural, and matches what I've heard in the real world.

Reviewer 2: It's the proper nown for a group of ministers in countries like the UK and NZ, it's a synonym for government. I was going to comment about whether any Americans would understand what it means.

Author: I assumed it was based on the Australian government which calls it "the Cabinet."

Reviewer 2: It's like saying Congress; "Who in Congress is blocking it?" "Who was just elected to Congress?"

Author: I'll leave this conversation unresolved hoping for feedback from people more in-the-know about Free Worlds political titles than us.

Reviewer 2: I'm pretty sure nobody knows what is what right now!

Author: I changed all references back to Cabinet without "the." Nobody is in "the cabinet" or "cabinet" now; those errors are corrected. Removing "the " takes out four characters.

> 			`	"The Hai have demanded we cut through the red tape by having the heads of state meet. They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`

Reviewer 2: ```suggestion
			`	"The Hai have demanded we cut through the red tape by having the heads of state meet. They insist on <first> as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`
```
I want to make it clear who he's talking about, it was left hanging before. (One more thing before you commit)

> 			`	"The Hai have demanded we cut through the red tape by having the heads of state meet. They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`

Reviewer 2: ```suggestion
			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns towards you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`
```
It could be done this way too.

Author: I used this option, but having "brief" twice in one sentence is confusing, so I also changed the last sentence to "We've gathered to prepare you for the meeting."

Reviewer 2: That is better, just needs one more thing.

> 			`	"The Hai have demanded we cut through the red tape by having the heads of state meet. They insist on <first> as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`

Reviewer 2: ```suggestion
			`	"The Hai have demanded we cut through the red tape by having the heads of state meet. They insist on <first> as a mediator." He turns towards you. "I think behind-the-scenes discussions have reached the point where we can proceed, and we've gathered to brief you before the meeting."`
```

> 			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns to you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to prepare you for the meeting."`

Reviewer 2: ```suggestion
			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns to you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed, and we've gathered to prepare you for the meeting."`
```
I think we need that sentence break there, or otherwise Xilin is stopping too much.

> 			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns to you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to prepare you for the meeting."`

Reviewer 2: ```suggestion
			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns to you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed, and we've gathered to prepare you for the meeting."`
```

				`	"What if we let them through, to attack the pirate planets?"`
					goto "let hai through"
				`	"What if we solve the kidnapping problem instead of the Hai doing it?"`
					goto "hai concerns"
				`	"Who in the Cabinet is blocking progress?"`
					goto "who is blocking"
			label "navy blocks hai"
			`	Xilin responds, "The Navy has blockaded the wormhole to the best of its ability, but the reality is that the Navy is already heavily committed, and barely look to be a match for the Hai fleet numbers. The Hai, on the other hand, do not yet appear to be committed heavily." He looks at Alondo briefly. "We had the unstated privilege of observing the Hai security fleet left behind on Everhope in addition to what we saw beyond the wormhole on our way in."`
			`	Danforth interjects, "In short, we doubt we can stop them while we're stretched out here... and if the reality is that they're only going to go after pirate worlds... well, we'd be under orders to disengage. The political crisis would be more tolerable than the probable military defeat."`
			`	Xilin nods grimly. "However, that would be a drastic failure of foreign policy."`
				goto questions
			label "let hai through"

> 			`	Xilin replies, "That would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood The Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`

Reviewer 2: ```suggestion
			`	Xilin replies, "That would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood the Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
```

				goto questions
			label "hai concerns"

> 			`	Xilin says, "They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in Cabinet disagree, that we're sort of at an impasse for the moment."`

Reviewer 2: ```suggestion
			`	Xilin says, "They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Cabinet is in disagreement about what to do, so there is deadlock at the moment."`
```
There is repetition here with "impasse", on line 1420.

> 			`	Xilin says, "They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree, that we're sort of at an impasse, for the moment."`

Reviewer 2: ```suggestion
			`	Xilin says, "They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree, that we're sort of at an impasse for the moment."`
```

				goto questions
			label "who is blocking"

> 			`	Xilin proceeds to list off a number of names, but after a few Turner cuts in again. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`

Reviewer 2: ```suggestion
			`	Xilin proceeds to list off a number of names, but to everyone's irritation and surprise, Turner cuts in. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`
```
I dunno if this works well, but it changes a part that doesn't make sense anymore.

Author: Yes, I need to revisit that section. Also, Captain Last needs to sound less robotic in her responses.

Author: With your changes, it does seem okay. (I always test these in game, not by reading the diffs, since everything feels different in the game than in a text editor.) I think the remaining issues with my A14 changes are that Captain Last is robotic.

			`	Teeneep begins, "I speak as the voice of the Hai and with the full authority of the Elder Council. Thousands of our people have been kidnapped over the past few decades, and they still largely reside under the servitude of various pirate factions that the Republic allows to continue. Even worse, your authorities have allowed numerous Hai to be slipped past on slaver ships, without attempting rescue because of the no boarding policy your forces employ, even on disabled and potentially doomed ships. We demand the return of our people and an immediate end to the policy of letting slaves die or escape without attempting boarding and rescue."`
			choice
				`	"How do the human governments respond?"`
			`	The President is the first to respond. "The problem of piracy in human space is a difficult one and cannot be solved overnight. The pirates are well-entrenched in their planets, and fighting them is a long and brutal struggle. With our navies distracted fighting one another, our recent war has left them stronger than ever."`

> 			`	The Chancellor continues, "The Free Worlds has made great strides reducing piracy by befriending the peoples of the anarchist worlds. We are discussing similar approaches with the Republic. Perhaps in another few years, piracy will be entirely eliminated."`

Reviewer 2: ```suggestion
			`	The Chancellor states, "However, the Free Worlds has made great strides reducing piracy by befriending the peoples of the anarchist worlds. We are discussing similar approaches with the Republic. Perhaps in another few years, piracy will be entirely eliminated."`
```

			choice
				`	"Are there any counter-proposals?"`
			`	"We have a counter-proposal," the President says. "We cannot afford to change our policy in totality, it would cost too many lives. However, we propose the following: The Free Worlds and Republic will establish a customs service on Mountaintop. Hai visiting human space will check in with us there. Our intelligence agencies will then track their movements in order to see to their safety as best we can. They'll also be given contacts on every human planet that they can go to for help. In the meantime, we will place extensive surveillance fleets at Ultima Thule to detect any unapproved Hai travel. We will do our best to disable and capture ships with kidnapped Hai, no matter the risk to Navy lives."`
			`	"The Free Worlds will also extend our protection, of course," the Chancellor adds.`
			`	Teeneep looks relieved by this suggestion.`
			choice
				`	"Is this an acceptable counter-proposal?"`

> 			`	Teeneep takes a moment to respond, "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`

Reviewer 2: ```suggestion
			`	Teeneep takes a moment to respond. "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
```

				goto questions
			label through
			action
				set "Hai Reveal [A14] A Time of Trouble: label through"
			`	Xilin replies, "That would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood the Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
				goto questions
			label concerns
			action
				set "Hai Reveal [A14] A Time of Trouble: label concerns"
			`	Xilin says, "They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. The Cabinet is in disagreement about what to do, so there is deadlock at the moment."`
				goto questions
			label blocking
			action
				set "Hai Reveal [A14] A Time of Trouble: label blocking"

> 			`	Xilin proceeds to list off a number of names, but to everyone's irritation and surprise, Turner cuts in. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`

Reviewer 2: ```suggestion
			`	Xilin proceeds to list off a number of names, but to everyone's irritation and surprise, Turner cuts in. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that, perhaps, if we can influence them to support a solution that we propose; even if we don't have that solution defined yet; the Hai could be a very large and very profitable market for Syndicate goods?"`
```
I'm never sure whether I'm the one who keeps suggesting all these commas that aren't correct.

Author: I generally accept any punctuation corrections, because I've gotten so many conflicting suggestions for punctuation changes that I've concluded I don't even know what punctuation is anymore.

			`	There is a long, expectant pause as Remington looks to Terry and they engage in some kind of silent exchange before he replies. "You know, I don't think we'd connected those two specific dots just yet." He looks at Terry. "We need to make some calls."`
				goto questions
			label begin
			`	A few hours later, they manage to set up a clear channel dedicated to the meeting in a room with large wall-mounted displays. You knew it was coming, and yet you're still a little surprised to see before you the President of the Republic and the Chancellor of the Free Worlds in remarkable clarity and surprisingly low latency. The windows behind the Chancellor suggest they were awoken in the middle of the night for this, but you wouldn't guess it from looking at them.`

> 			`	After a few seconds, you realize that three galactic leaders are waiting for you to begin the proceedings. As you've never been a galactic negotiator, you improvise the best you can.`

Reviewer 5: You should absolutely have been briefed by Xilin/Alondo in advance of this, even if just to say you've been given direction on what to expect and what is expected of you.

Author: I split up the first paragraph and added a brief by Alondo. Both individuals are sitting next to you, slightly off camera (similarly to how Teeneep has Sayari).

Reviewer 5: There's still the issue of you essentially improvising. This situation is far too sensitive for you to be put in that situation.

			choice
				`	"The tragedy we see around us can become a future of peace. Would the Hai representative please present us the problem we must solve together?"`
					goto formality
				`	"Teeneep, please explain the problem to the human leaders. I'm sure we can come to a compromise that will suit everyone."`
			`	Teeneep responds, "Thank you for mediating these proceedings, Captain <last>."`

> 			`	Both human leaders seem put off by the level of familiarity you just expressed, but Teeneep looks as if you took a world of stress off her shoulders.`

Reviewer 5: This sequence feels slightly gratuitous.

Author: I removed the "Both human leaders..." paragraph.

			label formality
			`	Teeneep explains, "I speak as the voice of the Hai and with the full authority of the Elder Council. Thousands of our people have been kidnapped over the past few decades, and they still largely reside under the servitude of various pirate factions that the Republic allows to continue. Even worse, your authorities have allowed numerous Hai to be slipped past on slaver ships, without attempting rescue because of the no boarding policy your forces employ, even on disabled and potentially doomed ships. We demand the return of our people and an immediate end to the policy of letting slaves die or escape without attempting boarding and rescue."`
			choice
				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleagues from the other side of the wormhole can help you solve this problem. President? Chancellor?"`
					goto formality2

> 				`	"Thank you, Teeneep. President, Chancellor, please explain the piracy situation from the human point of view."`

Reviewer 5: I think this choice is one too many.
Also, people at this level tend to only need to be addressed by title before immediately launching into their point of view ala:
Player: "President,"
President: <Immediately begins speaking.>

We should generally avoid asking these people to do or say the things that are the entire reason they're there for.

Author: Removed.

			`	The President seems surprised by the informal tone of your question, as if you just made a faux pas. The Chancellor looks more at ease than before, and perhaps a little amused.`
			label formality2
			`	The President is the first to respond. "The problem of piracy in human space is a difficult one and cannot be solved overnight. The pirates are well-entrenched in their planets, and fighting them is a long and brutal struggle. With our navies distracted fighting one another, our recent war has left them stronger than ever."`
			`	The Chancellor states, "However, the Free Worlds has made great strides reducing piracy by befriending the peoples of the anarchist worlds. We are discussing similar approaches with the Republic. Perhaps in another few years, piracy will be entirely eliminated."`
			choice

> 				`	"With respect, Chancellor, what you propose would take years to resolve a problem of immediate concern. How does the Hai representative respond?"`

Reviewer 5: The chancellor just said it would take years.
```suggestion
				`	"With respect, Chancellor, this is a problem of immediate concern. I'm sure the Hai representative would like to respond to that before we continue."`
```

					goto calm
				`	"Is this an acceptable proposal, Teeneep?"`

> 			`	Teeneep looks perturbed, and has to make an effort to hold back her anger and resentment. The other heads of state don't seem to notice; perhaps you're more sensitive to Hai facial expressions after the time you spent with the Hai. "Of course not," she says.`

Reviewer 6: ```suggestion
			`	Teeneep looks perturbed, and has to make an effort to hold back her anger and resentment. The other heads of state don't seem to notice; perhaps you're more sensitive to Hai facial expressions after the time you have spent with them. "Of course not," she says.`
```

			label calm
			`	Teeneep responds plainly, "We are a peaceful people by nature, but these pirate attacks have gone too far. We support a solution that benefits the innocent inhabitants of the pirate worlds. However, there must be immediate action, not years of delay."`
			choice
				`	"Surely a group of enlightened human leaders can propose an alternative?"`
				`	"Are there any counter-proposals?"`
					goto happypres

> 			`	The President seems genuinely irritated at being patronized, but the Chancellor smirks. You get a furry little smile from Teeneep.`

Reviewer 5: If the president is so easily read I suspect he wouldn't be president right now.
```suggestion
			`	The President presents a stony mask at being patronized, but the Chancellor smirks. You get a furry little smile from Teeneep.`
```

			label happypres
			`	"We have a counter-proposal," the President says. "We cannot afford to change our policy in totality, it would cost too many lives. However, we propose the following: the Free Worlds and Republic will establish a customs service on Mountaintop. Hai visiting human space will check in with us there. Our intelligence agencies will then track their movements in order to see to their safety as best we can. They'll also be given contacts on every human planet that they can go to for help. In the meantime, we will place extensive surveillance fleets at Ultima Thule to detect any unapproved Hai travel. We will do our best to disable and capture ships with kidnapped Hai, no matter the risk to Navy lives."`
			`	"The Free Worlds will also extend our protection, of course," the Chancellor adds.`
			`	Teeneep looks relieved by this suggestion.`
			choice

> 				`	"That sounds like a viable solution that will benefit all parties. How does the Hai representative respond?"`
> 				`	"Teeneep, will that satisfy the Hai?"`
> 					goto silence
> 			`	Both human leaders nod in agreement with you.`
> 			label silence
> 			`	Teeneep takes a moment to respond. "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`

Reviewer 5: We're belabouring the point here.
```suggestion
				`	"Would that be acceptable Elder Teeneep?"`
				`	"Teeneep, will that satisfy the Hai?"`
			`	Teeneep takes a moment to respond. "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
```

			`	The smoothness with which this goes down gives you the sense that the main points of this were all provisionally agreed to already, with only a little wiggle room left to be settled in this conversation.`
			`	Sayari steps up and whispers something to Teeneep who then seems to think for a moment before returning to the screens. "This pathway is acceptable."`
			choice
				`	"Thank you, President, Chancellor, and Hai representative, for a successful resolution."`
				`	"All of us should remember this day: the day three nations came together to form a lasting peace."`
					goto peace
				`	"Well, I guess I saved the galaxy yet again. Captain <last>, signing out!"`

> 					goto appalled

Reviewer 5: No.

Reviewer 2: Damn.

Author: Captain Last no longer signs out rudely. I left the other two closing statements.

			`	The President and Chancellor nod mechanically while Teeneep looks proud.`
				goto end
			label peace
			`	The President and Chancellor both look quite relieved, while Teeneep seems proud, and Sayari nods with approval.`
				goto end
			label appalled
			`	The President of the Republic looks appalled, while the Chancellor of the Free Worlds frowns with disapproval. Teeneep is annoyed, as if you ruined the moment. You can see Sayari in the background, shaking her head. A functionary near you gasps and drops her tablet, and it shatters audibly when it hits the ground.`
				goto end
			label end

> 			`	After the screens go off, a number of people involved in the busywork of bureaucracy start slipping out of the room, doubtlessly to get a head start on certain preparations as Teeneep and the leaders on the screen conclude some formal thanks for continued cooperation and other matters of protocol. You are too visible to slip away though and have to wait.`

Reviewer 2: ```suggestion
			`	After the screens go off, a number of people involved in the busywork of bureaucracy start slipping out of the room, doubtlessly to get a head start on certain preparations, as Teeneep and the leaders on the screen conclude some formal thanks for continued cooperation and other matters of protocol. You are too visible to slip away though and have to wait.`
```

			`	The President replies, "The Minister for Foreign Affairs has been keeping the Cabinet apprised of the major details and proposals reported back by Deputy Minister Yang. Under the circumstances we believe it would be appropriate for us to attempt a joint effort to address your concerns. We agree to making this effort against one target, and to reassessing the situation afterwards depending upon what is discovered in that process. The Republic representatives there with you now will work with you to identify that target and plan the action accordingly."`
			`	The smoothness with which this goes down gives you the sense that the main points of this were all provisionally agreed to already, with only a little wiggle room left to be settled in this conversation.`
			`	Sayari steps up and whispers something to Teeneep who then seems to think for a moment before returning to the screens. "This pathway is acceptable."`
			choice
				`	"Thank you, President, Chancellor, and Hai representative, for a successful resolution."`
				`	"All of us should remember this day: the day three nations came together to form a lasting peace."`

> 					goto peace

Reviewer 5: In my previous comment I was referring to this entire choice. 
None of these options are appropriate. In practice if anyone has a closing comment it should be the President.
If we need another break it should go later as you leave the room.

Author: I've changed it so Xilin is telling you the functionaries are asking for your signal to end the meeting. 

- Xilin whispers to you, "The functionaries are waiting for your signal to end the call so we can implement the decision. And, of course, so the Chancellor can get some sleep."

You can choose between two mundane responses:

- That ends these proceedings.
- Thank you, President, Chancellor, and Hai representative.

Teeneep's indication of pride is no longer so obvious:

- The President and Chancellor nod mechanically while Teeneep twists her ears and eyes in a way that usually signals pride.
